src/core/src/op/paged_attention.cpp

```diff
@@ -14,123 +14,125 @@ PagedAttentionExtension::PagedAttentionExtension(const ov::OutputVector& args) :
+                          "Rank of `key` input should be 2, but it is ",
+                          get_input_partial_shape(1).rank().get_length(),
+                          ".");
+    NODE_VALIDATION_CHECK(this,
+                          get_input_partial_shape(2).rank().is_dynamic() || get_input_partial_shape(2).rank().get_length() == 2,
+                          "Rank of `value` input should be 2, but it is ",
+                          get_input_partial_shape(2).rank().get_length(),
+                          ".");
 
-    // value: shape [batch_size, seq_len, num_kv_heads * head_size]
-    // auto value_type = get_input_element_type(2);
+    NODE_VALIDATION_CHECK(this,
+                          get_input_partial_shape(3).rank().is_dynamic() || get_input_partial_shape(3).rank().get_length() >= 2,
+                          "Rank of `key_cache` input should be at least 2, but it is ",
+                          get_input_partial_shape(3).rank().get_length(),
+                          ".");
+    NODE_VALIDATION_CHECK(this,
+                          get_input_partial_shape(4).rank().is_dynamic() || get_input_partial_shape(4).rank().get_length() >= 2,
+                          "Rank of `value_cache` input should be at least 2, but it is ",
+                          get_input_partial_shape(4).rank().get_length(),
+                          ".");
 
-    // is_prompt: boolean scalar
     NODE_VALIDATION_CHECK(this,
-                          // get_input_element_type(5) == ov::element::boolean &&
-                          get_input_shape(5) == ov::Shape({}),
-                          "is_prompt validation failed. ",
-                          "Got element type ",
+                          get_input_partial_shape(5).rank().is_dynamic() || get_input_partial_shape(5).rank().get_length() == 1,
+                          "Rank of `context_lens` input should be 1, but it is ",
+                          get_input_partial_shape(5).rank().get_length(),
+                          ".");
+    NODE_VALIDATION_CHECK(this,
+                          get_input_element_type(5).is_dynamic() || get_input_element_type(5) == element::i32,
+                          "Element type of `context_lens` input should be i32, but it is ",
                           get_input_element_type(5),
-                          ", shape ",
-                          get_input_shape(5));
-
-    // slot_mapping: shape [batch_size, max_context_len]
-    const auto& slot_mapping_shape = get_input_partial_shape(6);
+                          ".");
+    NODE_VALIDATION_CHECK(this,
+                          get_input_partial_shape(6).rank().is_dynamic() || get_input_partial_shape(6).rank().get_length() == 1,
+                          "Rank of `subsequence_begins` input should be 1, but it is ",
+                          get_input_partial_shape(6).rank().get_length(),
+                          ".");
```

Comment:
I think we should have a helper (like macro). This check is repeating many times here